### ISSUE TITLE
[Feature request] Add ability to remove local tags that are no longer on the remote repository

### ISSUE THREAD

**Issue by rventurelli** [ForkIssues/TrackerWin#1322]
It will be useful to clean remote tags that are no longer on the remote repository.
Following this [Stackoverflow question](https://stackoverflow.com/questions/1841341/remove-local-git-tags-that-are-no-longer-on-the-remote-repository/54297675) a possible solution can be to have an option on remote fetch to execute the following command:
```
git fetch --prune <remote> "+refs/tags/*:refs/tags/*"
```

**Comment by DanPristupov** [comment]
> ```
> git fetch --prune <remote> "+refs/tags/*:refs/tags/*"
> ```

That operation is dangerous because it removes ALL local tags which don't exist on the remote (even the new ones).

**Comment by gavignon** [comment]
I agree with this feature request, because currently I have to delete all local tags and then perform a git fetch again, I think that the fetch operation should give the option to prune local tags that are not on the remote.

**Comment by Zguy** [comment]
When you right-click on a remote there's a button called "Prune dead remote branches". It could be an option to add a button next to it "Prune tags not on remote" or similar. I feel like that would be less prone to people accidentally deleting tags they didn't intend to.

**Comment by BuzaL** [comment]
@DanPristupov , a "prune tags" would be nice, however the fetching should contain force by default (or at least be able to setup)
`git fetch --tags --force`
this line fetching and override all existing tags (most of the users using tags, not adding it), but new tags will remain untouched. (i found this as most relevant issue, but if you tell me, i will open a new one for it)

**Comment by BuzaL** [comment]
I will share what I using until - it might help others:
`git config --add remote.origin.fetch "+refs/tags/test_*:refs/tags/test_*"`

it override all the "test_" prefixed tags only. those are autogenerated and moving tags.
